### ISSUE TITLE
Use library versions

### ISSUE THREAD

**Issue by observant2** [maccam912/thewizzerofoz#1]
I just saw that the cargo.toml file contains:
```toml
[dependencies]
ggez = { git = "https://github.com/ggez/ggez" }
rand = "*"
find_folder = "*"
```
Meaning: "Always pull the newest version of every library". You should use constant versions, so your game won't break with new versions of ggez, rand and find_folder. :-)

**Comment by maccam912** [comment]
I did want to get back to you that I'm working on what you mentioned. I am using ggez 0.4.0 now and whatever the latest version of rand is right now (but not * any more, just in case). I got rid of find_folder since I don't seem to have it used anywhere, but there is a dependency of ggez 0.4.0 on app_dir, which is giving me an error:

the type of this value must be known in this context

Ok(Component::RootDir.as_ref().into()

Anyway no need to respond, I'll keep working on it, but since you are familiar enough with Rust to suggest setting specific versions, maybe you know of a way to either set an older version of app_dir here rather than what ggez 0.4.0 requires? Or am I out of luck because the version of app_dir ggez specifies just happens to not work for me?
